pkg/cmd/create/create_cluster_gke.go

```diff
@@ -592,20 +592,13 @@ func (o *CreateClusterGKEOptions) createClusterGKE() error {
 		return err
 	}
 
-	log.Logger().Info("Initialising cluster ...")
-
 	o.InstallOptions.SetInstallValues(map[string]string{
```

Comment:
ditto

Reply:
OK. Also, removing in /create_cluster_aws.go